romana-install/Vagrantfile

```diff
@@ -0,0 +1,92 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+require 'fileutils'
+
+$controller_vm_memory = 4096
+$worker_count = 1
+$worker_vm_memory = 1024
+
+CONFIG = File.expand_path("config.rb")
+if File.exist?(CONFIG)
+  require CONFIG
+end
+
+if $worker_vm_memory < 512
+  puts "Workers should have at least 512 MB of memory"
+end
+
+if $controller_vm_memory < 3072
+  puts "Controller should have at least 3072 MB of memory"
+end
+
+if $worker_count > 9
+  puts "Up to 9 workers supported only"
+  $worker_count = 9
+end
+
+Vagrant.configure(2) do |config|
+  # Common VM configuration
+  config.vm.box = "ubuntu/trusty64"
+  config.vm.box_url = "http://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-amd64-vagrant-disk1.box"
+  config.vm.synced_folder '../', '/home/vagrant/romana'
+  config.vm.provision "shell", inline: <<-EOF
+    if test -d /home/vagrant/romana/romana-install/deps/stack; then
+       if ! test -d /opt/stack; then
+         cp -a /home/vagrant/romana/romana-install/deps/stack /opt
+         chown -R ubuntu:ubuntu /opt/stack
+       fi
+    fi
+    test -d /home/vagrant/romana/romana-install/deps/archives && cp -a /home/vagrant/romana/romana-install/deps/archives/* /var/cache/apt/archives/
+    apt-get update && apt-get install -y git python-apt python-pip python-dev  python-netaddr && pip install awscli ansible
+  EOF
+  # ---
+
+  # Compute-specific VM configuration
+  (1..$worker_count).each do |i|
+    config.vm.define "romana-compute0#{i}" do |compute|
+      config.vm.hostname = "ip-192-168-99-1#{i}"

```

Comment:
The "99" here (and the rest of the file) isn't configurable, so they could only do an install with the default host_cidr.